### PR TITLE
[stable/kong] Improving docs to help when we set admin tls to false

### PR DESCRIPTION
**Pull request by marcostvz**
Improves a bit the documentation. When we set tls to false and change the ports, the readiness and liveness probes fail because it tries to connect with HTTPS to a HTTP port.

I was struggling with this when trying the new version and I believe it will help.

### CODE REVIEW

**Comment by marcostvz** [comment]
cc @shashiranjan84

File: stable/kong/Chart.yaml [hunk 1]
```diff
@@ -10,5 +10,5 @@ maintainers:
 name: kong
 sources:
 - https://github.com/Kong/kong
-version: 0.2.0
+version: 0.2.1
```
Comment (shashiranjan84) [inline stable/kong/Chart.yaml]: It conflicts with PR I sent for Kong v0.12.3

File: stable/kong/README.md [hunk 2]
```diff
@@ -69,7 +69,9 @@ and their default values.
 | env                               | Additional [Kong configurations](https://getkong.org/docs/latest/configuration/)               |
 | runMigrations                     | Run Kong migrations job                                                | `true`                |
 | readinessProbe                    | Kong readiness probe                                                   |                       |
+| readinessProbe.httpGet.schema     | Schema used by readyness probe. Set to HTTP if admin.useTLS = false    | `HTTPS`               |
```
Comment (cilindrox) [inline stable/kong/README.md]: Actual value [for these](https://github.com/kubernetes/charts/blob/master/stable/kong/values.yaml#L61) is `scheme` not _schema_ (`e` vs `a`)

File: stable/kong/Chart.yaml (hunk 1 above)
Comment (shashiranjan84) [reply stable/kong/Chart.yaml]: So you have to rebase once PR #4141 is merged

File: stable/kong/README.md (hunk 2 above)
Comment (marcostvz) [reply stable/kong/README.md]: True! 😱

File: stable/kong/README.md [hunk 3]
```diff
@@ -69,9 +69,9 @@ and their default values.
 | env                               | Additional [Kong configurations](https://getkong.org/docs/latest/configuration/)               |
 | runMigrations                     | Run Kong migrations job                                                | `true`                |
 | readinessProbe                    | Kong readiness probe                                                   |                       |
-| readinessProbe.httpGet.schema     | Schema used by readyness probe. Set to HTTP if admin.useTLS = false    | `HTTPS`               |
+| readinessProbe.httpGet.scheme     | Scheme used by readyness probe. Set to HTTP if admin.useTLS = false    | `HTTPS`               |
 | livenessProbe                     | Kong liveness probe                                                    |                       |
-| livenessProbe.httpGet.schema      | Schema used by liveness probe. Set to HTTP if admin.useTLS = false     | `HTTPS`               |
+| livenessProbe.httpGet.scheme      | Scheme used by liveness probe. Set to HTTP if admin.useTLS = false     | `HTTPS`               |
```
Comment (shashiranjan84) [inline stable/kong/README.md]: @marcostvz  personally I don't think we need to add this here, user can also set different test probe, like tcp probe or command probe. Warning on line 17 should be enough.

Comment (marcostvz) [reply stable/kong/README.md]: Good enough for me :) Removing it

File: stable/kong/values.yaml [hunk 4]
```diff
@@ -14,6 +14,7 @@ admin:
   #  service.beta.kubernetes.io/aws-load-balancer-proxy-protocol: "*"
 
   # HTTPS traffic on the admin port
+  # if set to false also set readinessProbe and livenessProbe httpGet scheme's to 'HTTP'
```
Comment (shashiranjan84) [inline stable/kong/values.yaml:17]: Sorry, should have recommended it before,

`# if set to false, you would also need to set both readinessProbe and livenessProbe httpGet scheme to 'HTTP'`

**Comment by carrodher** [comment]
It seems that in the meantime a new version has appeared #4290, can you increase the version again?

**Pull request merged**